README.md

```
@@ -1,3 +1,159 @@
+6 | 병 | ㅤ | 병 | ㅤ | 병 | ㅤ | 병 | ㅤ | 병 | 
+5 | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | 
+4 | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | 
+3 | ㅤ | 졸 | 졸 | ㅤ | 졸 | ㅤ | 졸 | ㅤ | 졸 | 
+2 | ㅤ | 포 | ㅤ | ㅤ | ㅤ | ㅤ | ㅤ | 포 | ㅤ | 
+1 | ㅤ | ㅤ | ㅤ | ㅤ | 왕 | ㅤ | ㅤ | ㅤ | ㅤ | 
+0 | 차 | 상 | 마 | 사 | ㅤ | 사 | 상 | 마 | 차 | 
+  ㅤ 0 ㅤ 1 ㅤ 2 ㅤ 3 ㅤ 4 ㅤ 5 ㅤ 6 ㅤ 7 ㅤ 8 
+
+[한의 차례입니다.]
+이동시킬 기물의 좌표를 입력해 주세요. 예) 03
+```
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)